src/libcore/num/f32.rs

```diff
@@ -161,6 +161,11 @@ impl f32 {
         self != self
     }
 
+    #[inline]
+    fn abs_(self) -> f32 {
+        f32::from_bits(self.to_bits() & 0x7fff_ffff)
```

Comment:
Why is this necessary, rather than using the existing `abs` method (which uses the LLVM `fabsf32` intrinsic directly)?

Reply:
That's only available in `std`. https://github.com/rust-lang/rust/issues/50145

Reply:
I'm bikeshedding here, but given typical naming conventions, I think it'd make sense to simply call this `abs` (which shouldn't cause any conflicts) and add a comment explaining why this method is private for discoverability, e.g.
```rust
// FIXME(#50145): `abs` is publicly unavailable in libcore due to concerns about
// about portability, so this implementation is for private use internally.
```

Reply:
How about `abs_private` with a comment?

Reply:
Added that.